src/libcore/slice/mod.rs

```diff
@@ -1541,6 +1541,9 @@ impl<T> [T] {
     /// let src = [1, 2, 3, 4];
     /// let mut dst = [0, 0];
     ///
+    /// // Note: the slices must be the same length, so you can slice the source
+    /// // or the destination to be the same size. Here we slice the source, four elements,
```

Comment:
> the slices must be the same length, so you can slice the source or the destination to be the same size

I spoke with a couple other documentation people and we agreed that this would be better worded as:

> Because the slices must be the same length, we'll need to slice the source slice from four elements to two. It will panic if we don't do this.

Reply:
Done.